http-agent/src/main/resources/db/migration/V1_MOTECH-2502.sql

```diff
@@ -0,0 +1,3 @@
+-- change type of field "responseBody" to text
+
+ALTER TABLE "HTTP_AGENT_MODULE_HTTPACTIONAUDIT" ALTER COLUMN "responseBody" TYPE text;
```

Comment:
From what I recall this was supposed to be mediumtext or longtext in mysql. Can you check what type is created on clean install w/o the migration when we set the field type as text and use that db type in migration?